p2p/net/session.go

```diff
@@ -25,6 +27,8 @@ type NetworkSession interface {
 	Encrypt(in []byte) ([]byte, error) // encrypt data using session enc key
 
 	EncryptGuard() *sync.Mutex // used for creating a per session transaction of data encryption and data delivery
+	Sign(data []byte) ([]byte, error)
```

Comment:
I believe we will have a lot of signing methods in our code, for better readability I'd recommend renaming this one to HmacSign

Reply:
In general I rather not expose internal impl. in methods names. If we indeed have more then one signature scheme (currently I don't see any reason for that) it will be probably be part of a different class which will be named appropriately